Applications/SimpleShot/layers/centering.cpp

```diff
@@ -0,0 +1,538 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2020 Jihoon Lee <jhoon.it.lee@samsung.com>
+ *
+ * @file   centering.cpp
+ * @date   08 Jan 2021
+ * @brief  This file contains the simple centering layer which has hardcoded
+ * mean feature vectors from given combinations
+ * @see    https://github.com/nnstreamer/nntrainer
+ * @author Jihoon Lee <jhoon.it.lee@samsung.com>
+ * @bug    No known bugs except for NYI items
+ *
+ */
+#include <iostream>
+#include <regex>
+#include <sstream>
+
+#include <nntrainer_error.h>
+#include <tensor.h>
+
+#include <centering.h>
+#include <simpleshot_utils.h>
+
+namespace simpleshot {
+namespace layers {
+
+constexpr float cifar100_resnet50[2048] = {
+  0.91757655,    1.5212023,     0.88224995,    1.5546391,     0.7752881,
```

Comment:
I think this needs to be model.bin for the network. need to deal with the way like weight. Also, it could differ depending on the feature network.

Reply:
I'll work on it.

Reply:
Done